V1/Controllers/WalletController.cs

```diff
@@ -27,9 +27,10 @@ public WalletController(IWalletService walletService)
         /// <param name="walletParameters"></param>
         /// <returns></returns>
         [HttpPost]
-        public async Task<IActionResult> CreateWallet(CreateWalletRequest walletParameters)
+        [Route("{sessionId}")]
+        public async Task<IActionResult> CreateWallet([FromBody] CreateWalletRequest walletParameters, string sessionId = null)
```

Comment:
[FromQuery] sessionId

Reply:
Also doesn't have to be defaulted to null, strings are automatically null if nothing is present.